src/Stripe.net/Services/Accounts/AccountBankAccountOptions.cs

```diff
@@ -3,11 +3,8 @@ namespace Stripe
 {
     using Newtonsoft.Json;
 
-    public class AccountBankAccountOptions : INestedOptions
+    public class AccountBankAccountOptions : INestedOptions, IHasObject
```

Comment:
This really feels wrong, the docstring of IHasObject is 

"Interface that identifies entities returned by Stripe that have an `object` attribute." and this is not an entity returned by Stripe.

Reply:
ah I guess the ship has sailed